registration/include/pcl/registration/impl/ia_fpcs.hpp

```diff
@@ -98,9 +101,12 @@ pcl::getMeanPointDensity (const typename pcl::PointCloud<PointT>::ConstPtr &clou
 
 #ifdef _OPENMP
 #pragma omp parallel for \
-  reduction (+:mean_dist, num) \
-  private (ids, dists_sqr) shared (tree, cloud, indices) firstprivate (s, max_dist_sqr) \
-  default (none)num_threads (nr_threads)
+  default(none) \
+  shared(tree, cloud, indices) \
+  private(ids, dists_sqr) \
```

Comment:
We might lose the information regarding need for only 2 elements in the vectors if the vectors are default initialized. I feel `private` should not be used, instead `firstprivate` should be used which has more defined semantics and intuitively correct usage. (All resources point to `firstprivate` being a specialization of `private`)

```suggestion
  firstprivate(ids, dists_sqr) \
```